Apps/WebClient/src/ClientApp/src/components/timeline/encounter.vue

```diff
@@ -75,9 +75,7 @@ $radius: 15px;
             <b-col>
                 <b-row>
                     <b-col cols="auto">
-                        <strong>
-                            Specialty Description:
-                        </strong>
+                        <strong> Specialty Description: </strong>
```

Comment:
Linter change?

Reply:
if it does, I like this new format better :-)

Reply:
yeah, updated the linter rules